settings/collect.ini

```diff
@@ -53,6 +53,13 @@ CollectAnonymousData=enabled
 CollectAnonymousDataList[]
 CollectAnonymousDataList[poll]=enabled
 CollectAnonymousDataAttribute=collection_anonymous
+# Are collected data sensitive? If true then user will be able to view collected
+# data only when previously submit a form
+CollectSensitiveData=true
+# Same as CollectSensitiveData but is a list with of IC types and
+# their override settings, if specified it will override default setting
+CollectSensitiveDataList[]
+CollectSensitiveDataList[poll]=false
```

Comment:
@glye According to old documentation https://doc.ez.no/eZ-Publish/Technical-manual/4.x/Reference/Content-classes/Content/Poll, `poll` content class is provided by us.